src/Sfa.Roatp.Indexer.ApplicationServices/IndexerService.cs

```diff
@@ -62,7 +62,7 @@ public IndexerService(IIndexSettings<T> indexSettings, IGenericIndexerHelper<T>
 
                     try
                     {
-                        await _indexerHelper.IndexEntries(newIndexName, providers).ConfigureAwait(false);
+                        _indexerHelper.IndexEntries(newIndexName, providers);
```

Comment:
The CheckRoatpAndCreateIndexAndUpdateAlias method is still marked async but has no await operators.